### ISSUE TITLE
Issue radio buttons - check boxes MATE

### ISSUE THREAD

**Issue by JulienNic0las** [vinceliuice/vimix-gtk-themes#45]
Hi,

I am not sure this issue is strictly Vimix related, so if it is not feel free to close this ticket.

I am using Mate 1.18.4 and installed the latest version of Vimix theme from source. I made sure that both pixbuf engine and murrine engine are installed with the right versions (so I am fairly confident that all requirements are satisfied). My OS is up to date (Solus). And I rebooted after installations. Everything works fine except radio buttons and check boxes which are displayed as plain red squares as it can be seen in the screenshot below.

![issue_boxes](https://user-images.githubusercontent.com/16461972/32405103-f8da31f6-c15e-11e7-9ffb-5a21eea3bb1c.png)

That said, keep up the good work. Vimix is a really nice looking theme.

rgds, 
Julien

**Comment by vinceliuice** [comment]
Use the install script to reinstall vimix, there must be something wrong with your installation, png files missed.

**Comment by JulienNic0las** [comment]
Thanks for the quick reply. In fact, I did install using the `Vimix-installer` script in root. I 've just tried again and the script ran to the end without any issues or warnings. But still, some images are missing.

However, for the sake of the completness, I have also tried to install the theme in local. This time checkboxes and radio buttons are correctely displayed. So it seems that this is rather an issue with copying some files in /usr/share/themes.

Thanks a lot for your support. I have to say that your theme is in my opinion better than the overrated adapta.

Bye
Julien
